dockerfiles/nginx/proxito.conf.template

```diff
@@ -82,6 +82,8 @@ server {
         add_header Access-Control-Allow-Headers $access_control_allow_headers always;
         set $x_frame_options $upstream_http_x_frame_options;
         add_header X-Frame-Options $x_frame_options always;
+        set $x_content_type_options $upstream_http_x_content_type_options;
+        add_header X-Content-Type-Options $x_content_type_options always;
```

Comment:
This change probably requires to be duplicated in corporate and also on -ops.

Reply:
Hrm... Good point. I'll check before merging.